src/init/features/dataconnect/index.ts

```diff
@@ -42,38 +48,24 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
       return { name: l, value: l };
     });
   }
-  const locationId = await promptOnce({
-    message: "What location would you like to deploy this service into?",
-    type: "list",
-    choices: locationOptions,
-  });
-  // TODO: Guided prompts to set up connector auth mode and generate
-  const connectorId = await promptOnce({
-    message: "What ID would you like to use for your connector?",
-    type: "input",
-    default: "my-connector",
-  });
-  const dir: string = config.get("dataconnect.source") || "dataconnect";
-  if (!config.has("dataconnect")) {
-    config.set("dataconnect.source", dir);
-    config.set("dataconnect.location", locationId);
-  }
+
   let cloudSqlInstanceId = "";
   let newInstance = false;
+  let locationId = "";
   if (setup.projectId) {
     const instances = await cloudsql.listInstances(setup.projectId);
-    const instancesInLocation = instances.filter((i) => i.region === locationId);
-    const choices = instancesInLocation.map((i) => {
-      return { name: i.name, value: i.name };
+    const choices = instances.map((i) => {
+      return { name: i.name, value: i.name, location: i.region };
     });
-    choices.push({ name: "Create a new instance", value: "" });
-    if (instancesInLocation.length) {
+    choices.push({ name: "Create a new instance", value: "", location: "" });
```

Comment:
Add an option with link to create the instance in cloud console instead.

nit: we can add a "Refresh the cloud sql instance list option" here for them to resume after clicking create button in console.

nit: add a TODO to only add "Create a free trial instance" if there isn't one already.

We can sort of predict whether they will run into free trial instance there.

Reply:
Oh, we already have `checkForFreeTrialInstance`.

Make it a blocking comment then. Let's move its logics before prompt for "Create a free trial instance"